afs/media/js/views/components/workflows/create-project-workflow/add-things-step.js

```diff
@@ -68,16 +68,16 @@ define([
         this.termOptions = [];
         this.value = ko.observableArray([]).extend({
             rateLimit: 100
-        });
-        this.startValue = ko.observableArray();
-        this.selectedResources = ko.observableArray([]);
-        this.addedValues = ko.observableArray();
-        this.removedValues = ko.observableArray();
+        });  // resourceinstanceid[]
+        this.startValue = ko.observableArray();  // Resource[]
+        this.selectedResources = ko.observableArray([]);  // Resource[]
+        this.addedValues = ko.observableArray();  // resourceinstanceid[]
+        this.removedValues = ko.observableArray();  // resourceinstanceid[]
```

Comment:
Then I clarified that here.

Reply:
This is interesting notation; makes me think of typescript.